README.md

```diff
@@ -657,12 +665,108 @@ Basic usage: `amclient.py <subcommand> [optional arguments]
     * `--ss-url <url>` - Storage Service URL (default: `http://127.0.0.1:8000`)
     * `--directory <dir>` - directory path in which to save the DIP
 
+* get-pipelines
+  * purpose: list (enabled) Pipelines known to the Storage Service.
+  * positional arguments:
+      * ss_api_key - Storage Service API key
+  * optional arguments:
+    * `--ss-user-name <username>` - Storage Service username (default: `test`)
+    * `--ss-url <url>` - Storage Service URL (default: `http://127.0.0.1:8000`)
+
+* get-transfer-status
+  * purpose: print the status of a transfer if it exists in a transfer
```

Comment:
Reifying "transfer workflow" sounds odd to me. Why are "transfer" and "ingest" workflows discussed here and below. Why not just say "if it exists in a pipeline"? This is probably overly nitpicky so ignore at your leisure.

Reply:
A good call.